install_dependencies.sh

```diff
@@ -125,13 +125,34 @@ cd c-ares-1.15.0
 make && make install
 cd ..
 
+if [[ "$OS_NAME" == "osx" ]]
+then
+  echo "installing nghttp2"
+  mkdir nghttp2
+  wget https://github.com/nghttp2/nghttp2/releases/download/v1.47.0/nghttp2-1.47.0.tar.gz -O ./nghttp2-1.47.0.tar.gz
```

Comment:
Suggest parametrizing version numbers to a variable so that we can more easily manager version increments later

Reply:
This is a good idea but not trivial for all the dependencies in the script...  For this ticket I think I'll skip it, but I did update the help message so all the libraries and versions are at least listed in one central place.